src/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-size: 0;
+}
+
+div {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-flow: column wrap;
+  width: 100px;
+}
```

Comment:
if you remove these styles, everything will still work fine

the idea here was to not only set the modifier, like `stars--3`, but also give that div the `stars` class. and set `display: flex` there. but you solved this problem using `display: inline-block;` for `stars__star`